agent/src/beerocks/fronthaul_manager/ap_manager/ap_manager_thread.cpp

```diff
@@ -132,30 +132,26 @@ bool ap_manager_thread::init()
         return false;
     }
 
-    //connect to slave //
+    return true;
+}
+
+void ap_manager_thread::connect_to_agent()
+{
+    if (slave_socket) {
+        LOG(WARNING) << "Agent socket is already open";
```

Comment:
IMO if slave socket is not nullptr, we should add the log but still restart the connection.

Reply:
I now see you only call this when slave_socket is `nullptr`. In this case, this would be more appropriate:
`LOG_IF(slave_socket, FATAL) << "Agent socket is already open!"`

Reply:
FATAL??? It is a little bit exaggerated don't you think?
FATAL is for serious system errors. This is just a wrong use of the function and therefore prints warning and returns with doing nothing.